### PR TITLE
fix: don't use deprecated currentlyFocusedField

### PR DESCRIPTION
**Pull request by rkdqudtjs1**
currentlyFocusedField => currentlyFocusedInput

### CODE REVIEW

File: packages/stack/src/views/KeyboardManager.tsx [hunk 1]
```diff
@@ -35,7 +35,8 @@ export default class KeyboardManager extends React.Component<Props> {
 
     this.clearKeyboardTimeout();
 
-    const input = TextInput.State.currentlyFocusedField();
+    // currentlyFocusedField => currentlyFocusedInput
```
Comment (satya164) [inline packages/stack/src/views/KeyboardManager.tsx:38]: Can you remove this comment?

Comment (rkdqudtjs1) [reply packages/stack/src/views/KeyboardManager.tsx]: I removed this comment.

File: packages/stack/src/views/KeyboardManager.tsx [hunk 2]
```diff
@@ -35,7 +35,8 @@ export default class KeyboardManager extends React.Component<Props> {
 
     this.clearKeyboardTimeout();
 
-    const input = TextInput.State.currentlyFocusedField();
+
+    const input = TextInput.State.currentlyFocusedInput();
```
Comment (satya164) [inline packages/stack/src/views/KeyboardManager.tsx:39]:
```suggestion
    const input = TextInput.State.currentlyFocusedInput
      ? TextInput.State.currentlyFocusedInput()
      : TextInput.State.currentlyFocusedField();
```

**Comment by ahilles107** [comment]
It will be backported to 4.x branch?

**Pull request merged**